app.js

```diff
@@ -1,19 +1,18 @@
 //jshint esversion:6
 
-require('dotenv').config();
-const express = require('express');
-const bodyParser = require('body-parser');
-const ejs = require('ejs');
-const gameRoutes = require('./routes');
+require("dotenv").config();
+const express = require("express");
+const bodyParser = require("body-parser");
+const ejs = require("ejs");
+const gameRoutes = require("./routes");
 
 const app = express();
 
-app.set('view engine', 'ejs');
-app.use(bodyParser.urlencoded({extended: true}));
-app.use(express.static(__dirname + '/public'));
+app.set("view engine", "ejs");
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + "/public"));
 app.use(gameRoutes);
 
-app.listen(3000, function(){
-    console.log("Server is Running on Port 3000");
```

Comment:
change port in console.log message

Reply:
I have updated the port in ap.js

Reply:
I am making a final commit again changing both the port and message to 4000

Reply:
Updated